lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -192,6 +212,10 @@ function dosomething_northstar_transform_user($user, $password = null) {
 
   // Set values in ns_user if they are set.
   foreach ($optional as $ns_key => $drupal_key) {
+    if (! isset($user->$drupal_key)) {
```

Comment:
Added this because _currently_ Facebook users dont have birthdays being added, was throwing warnings